packages/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -46,6 +44,34 @@ const useStyles = makeStyles({
   outOfOfficeAway: theme => ({
     color: theme.colorPaletteMarigoldBackground3,
   }),
+
+  // Icons are not resizeable, and these sizes are currently missing
+  // use `!important` to size the currently available icons to the missing ones
+  //
+  tiny: {
+    aspectRatio: '1',
+    width: '6px',
+    '& svg': {
+      width: '6px !important',
+      height: '6px !important',
+    },
```

Comment:
Presumably this `!important` will go away when we have resizable icons?

Reply:
yes exactly